src/components/form/Wrapper.styled.js

```diff
@@ -26,6 +26,7 @@ const Wrapper = styled.div`
 		border-radius: 5px;
 		margin: 0.5rem;
 		margin-left: 1.1rem;
+		margin-right: 1.1rem;
 		width: 16.25rem;
```

Comment:
setting the with of an object to a rem size might lead to some accessibility problems. When a user sets the font size of the browser to 200-300% of the standard size (which is common for users with restricted vision) your element might become wider than 100vw. The best would be to let the parent element of the wrapper decide how much space the wrapper gets (with a grid for example). If that is not possible, you can use a fixed pixel value here.